opm/output/Wells.hpp

```diff
@@ -150,7 +176,43 @@ namespace Opm {
     inline double& Rates::get_ref( opt m ) {
         return const_cast< double& >(
                 static_cast< const Rates* >( this )->get_ref( m )
-               );
+                );
+    }
+
+    inline Well& Wells::operator[]( const std::string& k ) {
```

Comment:
Maybe. It exists for clients so they have an easier time building in the case of many wells. I have been playing with the idea of forbidding to modify an established `Wells` though, i.e. all wells (and by extension connections) must be passed at construction time.